mappings/net/minecraft/world/gen/foliage/FoliagePlacer.mapping

```diff
@@ -6,11 +6,55 @@ CLASS net/minecraft/class_4647 net/minecraft/world/gen/foliage/FoliagePlacer
 		ARG 1 radius
 		ARG 2 randomRadius
 		ARG 3 type
+	METHOD method_23447 getRadiusForPlacement (IIII)I
+		ARG 1 trunkHeight
+		ARG 2 baseHeight
+			COMMENT This method is used to ensure that a tree can place foliage when it generates.
+			COMMENT
+			COMMENT It runs for every y-level of the tree being generated.
+			COMMENT
+			COMMENT @param trunkHeight the height of the trunk
+			COMMENT @param baseHeight the height of the hull tree
+			COMMENT @param radius the radius of the FoliagePlacer
+			COMMENT @param currentTreeHeight the current y-level of the tree being tested.
```

Comment:
Why are you commenting on param 2 than the method itself? also `@param` tags should be replaced by comments on individual parameter themselves.

Reply:
oops